src/main/java/io/gravitee/cockpit/api/command/v1/targettoken/TargetTokenReply.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.cockpit.api.command.v1.targettoken;
+
+import io.gravitee.cockpit.api.command.v1.CockpitCommandType;
+import io.gravitee.cockpit.api.command.v1.CockpitReply;
+import io.gravitee.exchange.api.command.CommandStatus;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
+public class TargetTokenReply
+  extends CockpitReply<TargetTokenReplyPayload<String>> {
+
+  private String targetToken;
```

Comment:
I'm not sure if required right now, but what do you think about adding a way to identify the token on APIM side?
In case we need to remove it, for example

Reply:
I use the id sent in TargetTokenCommandPayload to set the source id of the user created in apim like we do for user command handler. So, I was thinking in the future for revoke we can find the apim user from the source id and delete the user along with any tokens it has. What do you think of that approach?

Reply:
even better!